lib/kredits.js

```diff
@@ -1,7 +1,7 @@
 const ethers = require('ethers');
 const RSVP = require('rsvp');
 
-const Healthcheck = require('./utils/healthcheck');
+const PreflightChecker = require('./utils/preflight_checker');
```

Comment:
Why not just `Preflight.check()` if there's a checker method in that module anyway? I think it would read/look a bit nicer.